index.js

```diff
@@ -0,0 +1,16 @@
+const express = require('express')
+const teams = require('./teams')
+
+const app = express()
+
+app.get('/teams', (request, response) => {
+  return response.send(teams)
+})
+app.get('/teams', (request, response) => {
+  return response.send(teams.id)
+})
```

Comment:
Teams is an array of objects so you can't `teams.id`. You need to capture the id the user wants in the route using the `/teams/:id` syntax to create the parameter. Whatever they type will go into an variable called `id` which is stored in `request.params`. So if the user goes to `/teams/12` then `request.params.id` will be `12`. You can then use that number to find the right team object in the teams array with something like find.